README.md

```diff
@@ -80,24 +82,16 @@ Finding the longest movie is almost impossible using that format, so let's chang
 }
 ```
 
-Should be:
-
-```javascript=
-{
-  "title":"The Shawshank Redemption",
-  "year":"1994",
-  "director":"Frank Darabont",
-  "duration":"142",
-  "genre":["Crime","Drama"],
-  "rate":"9.3"
-}
+...would be turned into:
+```javascript
+"The Shawshank Redemption by Frank Darabont (1994)"
 ```
- 
-You must return a new array with all the info about the movie, not modify the original array!
+
+**:bulb: Remember we have a methods to create a new array from another one! This method is `map`.**
```

Comment:
Suggested update:

**:bulb: Remember we have methods to create a new array without making changes to the original array! One of these methods is `map`.**